dts/bindings/rtc/rtc.yaml

```diff
@@ -0,0 +1,31 @@
+---
+title: RTC Base Structure
+id: rtc
+version: 0.1
+
+description: >
+    This binding gives the base structures for all RTC devices
+
+
+properties:
+    clock-frequency:
+      type: int
+      category: optional
+      description: Clock frequency information for RTC operation
+      generation: define
+    label:
+      type: string
+      category: required
+      description: Human readable string describing the device (used by Zephyr for API name)
+      generation: define
+    interrupt-names:
```

Comment:
Why are interrupt-names part of the base RTC structure, but not interrupts?

Reply:
Now fixed in https://github.com/zephyrproject-rtos/zephyr/pull/7182/commits/0fc81773e63f439238ab803e04eb55aa90dec4ab